src/Sentry.EntityFramework/Sentry.EntityFramework.csproj

```diff
@@ -1,11 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>net6.0;net462;netstandard2.1</TargetFrameworks>
     <Description>Official Entity Framework 6 integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
     <PackageTags>$(PackageTags);EntityFramework;EF;EF6</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(FrameworkSupportsAot)' == 'true'">
```

Comment:
Is there any reason why we must define this conditionally instead of just for all frameworks?

Reply:
Under the hood it sets things like `IsTrimmable`. I don't think you can do that on a .NET Framework application or a .NET 5.0 application as it was only introduced in .NET 6.

Our Device tests fall over if we enable it for `net7.0-android` as well... so currently we can't turn this on for all frameworks.

Reply:
You can sat whatever properties you want. They won't just have any effect.

Reply:
> Our Device tests fall over if we enable it for `net7.0-android` as well... so currently we can't turn this on for all frameworks.

I was talking specifically about the added property, i.e. `<IsAotCompatible>`. Does it trip tests or can we just add it in general?

Reply:
I mean I don't mind this too much, it's just that I prefer avoiding conditional properties unless we know they're necessary, otherwise it makes reasoning about things harder (for me).

Reply:
> I was talking specifically about the added property, i.e. `<IsAotCompatible>`. Does it trip tests or can we just add it in general?

Yeah, that was the reason I added conditions like targetframework != net7.0-android... in that case, it compiles, but our tests fail (it is [still experimental](https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/?tabs=net8plus%2Cwindows#platformarchitecture-restrictions) there I think). 

From memory, if you try to enable IsTrimmable for all the frameworks, the compiler bleats for .net framework and netcoreapp3.1... but it was such a game of wack-a-mole, my memory may not be very accurate. I think my brain blocked some of it out as traumatic. 

Regardless, until the issues with android are resolved, I don't think we can enable it unconditionally. At least we have a single place where we can modify the conditionality though.

Reply:
thx for the explanatino